doc/manual/src/installation/installing-binary.md

```diff
@@ -1,89 +1,112 @@
 # Installing a Binary Distribution
 
-The easiest way to install Nix is to run the following command:
+To install the latest version Nix, run the following command:
 
 ```console
 $ curl -L https://nixos.org/nix/install | sh
 ```
 
-This will run the installer interactively (causing it to explain what
-it is doing more explicitly), and perform the default "type" of install
-for your platform:
-- single-user on Linux
-- multi-user on macOS
+This will run the installer interactively, that will explain what it is doing.
+It will perform the default type of installation for your platform:
 
-  > **Notes on read-only filesystem root in macOS 10.15 Catalina +**
-  >
-  > - It took some time to support this cleanly. You may see posts,
-  >   examples, and tutorials using obsolete workarounds.
-  > - Supporting it cleanly made macOS installs too complex to qualify
-  >   as single-user, so this type is no longer supported on macOS.
+- [Multi-user](#multi-user-installation):
+  - Linux with systemd and without SELinux
+  - macOS
+- [Single-user](#single-user-installation):
+  - Linux without systemd
+  - Linux with SELinux
 
-We recommend the multi-user install if it supports your platform and
-you can authenticate with `sudo`.
+We recommend the multi-user installation if it supports your platform and you can authenticate with `sudo`.
 
-# Single User Installation
-
-To explicitly select a single-user installation on your system:
+The installer can configured with various command line arguments and environment variables.
+To show available command line flags:
 
 ```console
-$ curl -L https://nixos.org/nix/install | sh -s -- --no-daemon
+$ curl -L https://nixos.org/nix/install | sh -s -- --help
 ```
 
-This will perform a single-user installation of Nix, meaning that `/nix`
-is owned by the invoking user. You can run this under your usual user
-account or root. The script will invoke `sudo` to create `/nix`
-if it doesn’t already exist. If you don’t have `sudo`, you should
-manually create `/nix` first as root, e.g.:
+To check what it does and how it can be customised further, [download and edit the second-stage installation script](#installing-from-a-binary-tarball).
+
+# Installing a pinned Nix version from a URL
+
+Version-specific installation URLs for all Nix versions since 1.11.16 can be found at [releases.nixos.org](https://releases.nixos.org/?prefix=nix/).
+The corresponding SHA-256 hash can be found in the directory for the given version.
+
+These installation scripts can be used the same as usual:
 
 ```console
-$ mkdir /nix
-$ chown alice /nix
+$ VERSION=2.19.2 curl -L https://releases.nixos.org/nix/nix-$VERSION/install | sh
```

Comment:
At least in POSIXy shells, I think these have to get split up across two lines if it's going to work. (Or keep telling the user to manually write it in as the previous text did).

Parameter assignments prefixed to an invocation will be exported to the environment of the running command right before it's executed, but  parameter expansion happens very early on in the process of evaluating input from the user:

```
$ bash -c 'VERSION=2.19.2 echo curl -L https://releases.nixos.org/nix/nix-$VERSION/install'
curl -L https://releases.nixos.org/nix/nix-/install

$ zsh -c 'VERSION=2.19.2 echo curl -L https://releases.nixos.org/nix/nix-$VERSION/install'
curl -L https://releases.nixos.org/nix/nix-/install

$ fish -c 'VERSION=2.19.2 echo curl -L https://releases.nixos.org/nix/nix-$VERSION/install'
curl -L https://releases.nixos.org/nix/nix-2.19.2/install

$ dash -c 'VERSION=2.19.2 echo curl -L https://releases.nixos.org/nix/nix-$VERSION/install'
curl -L https://releases.nixos.org/nix/nix-/install
```

Reply:
Ah yes

Reply:
```suggestion
$ export VERSION=2.19.2 
$ curl -L https://releases.nixos.org/nix/nix-$VERSION/install | sh
```